brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1970 Slider block basic

### Description
Slider Block basic functionality ( Add, remove, save slides )

### Types of changes
New feature (non-breaking change which adds functionality) 

### How has this been tested?
- Go to block inserter in the editor 
- Search for the slider block, and select it.
- By default, only 2 slides are added. You can add more slides by clicking on Add slides icon from the Top toolbar 
- You can remove slides from the list view. 
- Try adding and removing slides. The sequence of slides should work as expected.
- Try adding blocks to individual slides and save changes. Make sure the changes are saved and working as expected. 
- General and style options for the block are not working now. Those will be covered in separate PR. 

### Checklist:
